src/activities/categorization/Categorization.qml

```diff
@@ -0,0 +1,229 @@
+import "qrc:/gcompris/src/core/core.js" as Core
+import "."
+
+ActivityBase {
+    id: activity
+    
+    onStart: focus = true
+    onStop: {}
+    
+    property string boardsUrl: "qrc:/gcompris/src/activities/categorization/resource/"
+    property string imagesUrl: "qrc:/gcompris/src/activities/categorization/resource/images"
+    property bool vert: background.width < background.height
+    
+    pageComponent: Image {
+        id: background
+        source: activity.boardsUrl +"imageid-bg.svg"
+        anchors.fill: parent
+        sourceSize.width:parent.width
+        signal start
+        signal stop
+        
+        Component.onCompleted: {
+            activity.start.connect(start)
+            activity.stop.connect(stop)
+        }
+        
+        // Add here the QML items you need to access in javascript
+        QtObject {
+            id: items
+            property Item main: activity.main
+            property alias background: background
+            property alias bar: bar
+            property alias bonus: bonus
+            property alias categoryReview: categoryReview
+            property alias menuScreen: menuScreen
+            property alias menuModel: menuScreen.menuModel
+            property alias dialogActivityConfig: dialogActivityConfig
+            property bool easyMode:true
+            property bool mediumMode: false
+            property bool expertMode: false
+            property bool instructionsChecked: true
+            property bool categoryNameChecked: true
+            property bool categoryImageChecked: true
+            property bool scoreChecked: true
+            property var details
+        }
+        
+        onStart: { 
+            Activity.init(items,boardsUrl)
+            dialogActivityConfig.getInitialConfiguration()
+            Activity.start()
+        }
+        
+        onStop: {
+            dialogActivityConfig.saveDatainConfiguration() 
+        }
+        
+        MenuScreen {
+            id: menuScreen
+        }
+        
+        CategoryReview {
+            id: categoryReview
+        }
+        
+        DialogActivityConfig {
+            id: dialogActivityConfig
+            content:Component{
+                Flow {
+                    spacing: 5
+                    width: dialogActivityConfig.width
+                    height: dialogActivityConfig.height
+                    
+                    GCDialogCheckBox {
+                        id: easyModeBox
+                        width: 250 * ApplicationInfo.ratio
+                        text: qsTr("Easy mode")
+                        checked: items.easyMode
+                        onCheckedChanged: {
+                            print("Easy Mode: ",checked)
+                            if(items.expertMode == true){
+                                items.expertMode = false
+                                expertModeBox.checked = false
+                                items.instructionsChecked = true
```

Comment:
you can probably bind them directly in the qml, for example: instructionsChecked: (mode === "easy" || mode === "medium")